client/components/Profile.jsx

```diff
@@ -34,24 +38,51 @@ class Profile extends React.Component {
               <fieldset>
                 <legend>Profile</legend>
 
-                <label htmlFor='username'>Username</label>
+                {/* <img src={profilePic} alt={name}> </img> */}
```

Comment:
Are we keeping this here for a good reason?

Reply:
Yes, for users to upload images (eventually)